languages/en-US/moderation/setwarns.json

```
@@ -2,11 +2,11 @@
     "DESCRIPTION": "Define the sanctions that members will get after a certain number of warns!",
     "USAGE": "{{prefix}}setwarns [kick/ban] [number/reset]",
     "EXAMPLES": "{{prefix}}setwarns kick 10\n{{prefix}}setwarns ban 10\n{{prefix}}setwarns ban reset",
-    "MISSING_TYPE": "Please specify a valid sanction type! (`kick`, `ban`)",
-    "SUCCESS_KICK": "Configuration saved! When a member will reach **{{count}}** warns, they will be kicked. Type `{{prefix}}configuration` to see your new configuration!",
-    "SUCCESS_BAN": "Configuration saved! When a member will reach **{{count}}** warns, he will be banned. Type `{{prefix}}configuration` to see your new configuration!",
-    "SUCCESS_KICK_RESET": "Configuration saved! The kick sanction is no longer automatic! Type `{{prefix}}configuration` to see your new configuration!",
-    "SUCCESS_BAN_RESET": "Configuration saved! The ban sanction is no longer automatic! Type `{{prefix}}configuration` to see your new configuration!",
-    "AUTO_BAN": "**{{username}}** was automatically banned because they reach more than **{{count}}** warns!",
-    "AUTO_KICK": "**{{username}}** was automatically kicked because they reach more than **{{count}}** warns!"
+    "MISSING_TYPE": "Please specify sanction between `kick` and `ban`",
+    "SUCCESS_KICK": "Configuration saved! When a member will reach **{{count}}** warns, he will be kicked.\nType `{{prefix}}configuration` to see your new configuration!",
+    "SUCCESS_BAN": "Configuration saved! When a member will reach **{{count}}** warns, he will be banned.\nType `{{prefix}}configuration` to see your new configuration!",
+    "SUCCESS_KICK_RESET": "Configuration saved! The `kick` sanction is no longer automatic!\nType `{{prefix}}configuration` to see your new configuration!",
+    "SUCCESS_BAN_RESET": "Configuration saved! The `ban` sanction is no longer automatic!\nType `{{prefix}}configuration` to see your new configuration!",
+    "AUTO_BAN": "**{{username}}** was automatically banned because he/she reach more than **{{count}}** warns!",
+    "AUTO_KICK": "**{{username}}** was automatically kicked because he/she reach more than **{{count}}** warns!"
```

Sauf erreur de ma part, le they singulier correspond parfaitement à ce cas (https://fr.wikipedia.org/wiki/They_singulier)
```suggestion
    "AUTO_BAN": "**{{username}}** was automatically banned because he/she reach more than **{{count}}** warns!",
    "AUTO_KICK": "**{{username}}** was automatically kicked because he/she reach more than **{{count}}** warns!"
```

---

Ca me va, c'est engagé en plus 👍

---

b535d6f82539c53002e1613fcd3690af22861f41